### PR TITLE
Issue #5603 - Single page documentation

### PR DESCRIPTION
**Pull request by WalkerWatch**
Issue #5603 Creates single page output for 9.4.x documentation and aligns with 10 and 11 documentation formatting.

### CODE REVIEW

**Comment by jmcc0nn3ll** [comment]
If we are going to wack out the docbook backend entirely we should clear out the xslt files, basically the src/main/docbkx-*  directories.  Might be good to go through and make sure none of that is referenced but I think that stuff was mostly just syntax highlighter and font type stuff.

**Review by jmcc0nn3ll — CHANGES_REQUESTED** [review]
See prior comment.

File: jetty-documentation/pom.xml [hunk 1]
```diff
@@ -1,145 +1,60 @@
...
-              </resources>
-              <outputDirectory>${html.directory}</outputDirectory>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
       <plugin>
         <groupId>org.asciidoctor</groupId>
         <artifactId>asciidoctor-maven-plugin</artifactId>
+        <version>1.5.7.1</version>
```
Comment (sbordet) [inline jetty-documentation/pom.xml]: No.
The version should be inherited from the root pom, which is already at 2.1.0.

File: jetty-documentation/pom.xml [hunk 2]
```diff
@@ -1,145 +1,60 @@
...
+        <dependencies>
+          <dependency>
+            <groupId>org.asciidoctor</groupId>
+            <artifactId>asciidoctorj-diagram</artifactId>
+            <version>2.0.5</version>
+          </dependency>
+        </dependencies>
```
Comment (sbordet) [inline jetty-documentation/pom.xml]: No. Asciidoctor dependencies should be declared in the root pom, so we keep the versions of those dependencies up-to-date as well.

**Review by sbordet — CHANGES_REQUESTED** [review]
I ran the documentation generation, and I get a ton of warnings and errors.

In particular, all source code blocks for XML are broken, so the generated documentation is unreadable.

Remove `subs={sub-order}` from everywhere, it's now broken, it's unnecessary in most cases, and should be explicitly added only in blocks that require it.

The "parts" use level 0 sections, should be using level 1 (see warning during generation).

There is also a number of invalid cross references that should be fixed.

File: jetty-documentation/src/main/asciidoc/administration/alpn/alpn.adoc [hunk 3]
```diff
@@ -90,7 +90,7 @@ or alternatively you need the link:#alpn-openjdk8-agent[Jetty ALPN agent].
 
 In the case of the Jetty ALPN boot library, start the JVM as follows:
 
-[source, plain, subs="{sub-order}"]
+[source, plain, ]
```
Comment (sbordet) [inline jetty-documentation/src/main/asciidoc/administration/alpn/alpn.adoc]: Ah, no: the last comma must be removed as well. I know it renders without issues, but it's not good as it looks like there is something missing.

**Review by sbordet — CHANGES_REQUESTED** [review]

File: jetty-documentation/src/main/asciidoc/administration/alpn/alpn.adoc (hunk 3 above)
Comment (WalkerWatch) [reply jetty-documentation/src/main/asciidoc/administration/alpn/alpn.adoc]: Yeah, I thought about this after I committed. Fixing now.

**Review by jmcc0nn3ll — APPROVED** [review]

**Comment by jmcc0nn3ll** [comment]
As long as this visually looks fine I am good to go with it, it lets us get the updated website leveraging it so we can deploy that going into the 10 and 11 releases.  Any content changes we need to make can be done separately.

**Comment by WalkerWatch** [comment]
@sbordet Any other changes you'd like to see?

File: jetty-documentation/src/main/asciidoc/administration/http2/configuring-haproxy.adoc [hunk 4]
```diff
@@ -113,7 +113,7 @@ The `domain.pem` file will be used later by HAProxy.
 Now we can setup `haproxy.cfg` to configure HAProxy.
 This is a minimal configuration:
 
-[source, ,subs="{sub-order}"]
+[source, ,]
```
Comment (sbordet) [inline jetty-documentation/src/main/asciidoc/administration/http2/configuring-haproxy.adoc]: Fix this.

File: jetty-documentation/src/main/asciidoc/configuring/security/configuring-form-size.adoc [hunk 5]
```diff
@@ -35,7 +35,7 @@ There exists 2 system properties that will adjust the default maximum form sizes
 
 Used from command line as such:
 
-[source,shell,subs="{sub-order}"]
+[source,shell,]
```
Comment (sbordet) [inline jetty-documentation/src/main/asciidoc/configuring/security/configuring-form-size.adoc]: Fix this.

File: jetty-documentation/src/main/asciidoc/configuring/security/configuring-form-size.adoc [hunk 6]
```diff
@@ -45,7 +45,7 @@ $ java -Dorg.eclipse.jetty.server.Request.maxFormContentSize=400000 -jar ...
 
 Or via Java code (make sure you do this before you instantiate any `ContextHandler`, `ServletContextHandler`, or `WebAppContext`)
 
-[source,java,subs="{sub-order}"]
+[source,java,]
```
Comment (sbordet) [inline jetty-documentation/src/main/asciidoc/configuring/security/configuring-form-size.adoc]: Fix this.

File: jetty-documentation/src/main/asciidoc/configuring/security/configuring-form-size.adoc [hunk 7]
```diff
@@ -56,7 +56,7 @@ System.setProperty(ContextHandler.MAX_FORM_CONTENT_SIZE_KEY, "400000");
 
 To configure the form limits for a single web application, the context handler (or webappContext) instance must be configured using the following methods:
 
-[source,java,subs="{sub-order}"]
+[source,java,]
```
Comment (sbordet) [inline jetty-documentation/src/main/asciidoc/configuring/security/configuring-form-size.adoc]: Fix this.

File: jetty-documentation/src/main/asciidoc/quick-start/getting-started/jetty-running.adoc [hunk 8]
```diff
@@ -25,7 +25,7 @@ Jetty has no GUI (Graphical User Interface), so running the server and performin
 Once you have access to your system's command line, navigate to the directory where you unpacked your copy of the Jetty distribution.
 To start Jetty on the default port of 8080, run the following command:
 
-[source,screen,subs="{sub-order}"]
+[source,screen,]
```
Comment (sbordet) [inline jetty-documentation/src/main/asciidoc/quick-start/getting-started/jetty-running.adoc]: Fix this.

**Comment by WalkerWatch** [comment]
@sbordet Done, not sure how I managed to miss those. CI Building now. Good to go when green?

**Review by sbordet — APPROVED** [review]

**Review by sbordet — APPROVED** [review]

**Pull request merged**